Tease AI/Form1/ScriptVariables.vb

```diff
@@ -0,0 +1,284 @@
+﻿'===========================================================================================
+'
+'									 ScriptVariables.vb.vb
+'
+' This file contains functions and methods to handle script Variables.
+' 
+'===========================================================================================
+Imports System.IO
+
+Partial Public Class Form1
+
+	Public Function VariableFolder() As String
+		VariableFolder = Application.StartupPath & "\Scripts\" & dompersonalitycombobox.Text & "\System\Variables\"
+		Directory.CreateDirectory(VariableFolder)
+	End Function
+
+	Public Function VariableExists(varName As String)
+
+		Return File.Exists(Path.Combine(VariableFolder, varName))
+
+	End Function
+
+
+	Public Sub SetVariable(ByVal varName As String, ByVal value As String)
+
+		My.Computer.FileSystem.WriteAllText(Path.Combine(VariableFolder, varName), value, False)
+
+	End Sub
+
+	Public Function GetVariable(ByVal varName As String) As String
+
+		If VariableExists(varName) Then
+			Return TxtReadLine(Path.Combine(VariableFolder, varName))
+		Else
+			Return 0
+		End If
+
+	End Function
+
+	Public Sub DeleteVariable(ByVal varName As String)
+
+		Dim filePath As String = Path.Combine(VariableFolder, varName)
+		If File.Exists(varName) Then File.Delete(filePath)
```

Comment:
Guessing this should be:
    If File.Exists(filePath) Then File.Delete(filePath)

Reply:
You are guessing right. I've pushed your suggested changes.
Thanks for reviewing and telling!